docs/source/usage/cli.rst

```diff
@@ -170,11 +170,15 @@ Image
 
 Image-based provisioning can be accomplished through the use of `Metalsmith`_. It requires the image to be uploaded into OpenStack Glance. Once that's done, a non-admin can run the following:
 
-+----------------+---------------------------------------------------------------------------------------------------------------------+
-|                | **Actions**                                                                                                         |
-+----------------+---------------------------------------------------------------------------------------------------------------------+
-| Provision Node | ``metalsmith deploy --resource-class baremetal --image <image> --network <network> --ssh-public-key <path-to-key>`` |
-+----------------+---------------------------------------------------------------------------------------------------------------------+
++-------------------+---------------------------------------------------------------------------------------------------------------------+
+|                   | **Actions**                                                                                                         |
++-------------------+---------------------------------------------------------------------------------------------------------------------+
+| Provision Node    | ``metalsmith deploy --resource-class baremetal --image <image> --network <network> --candidate <node-uuid> --ssh-public-key <path-to-key>`` |
```

Comment:
I'm pretty sure this spacing breaks the generated table; you can test this on your local machine by  creating a virtualenv and installing the requirements, going into the doc/ directory, and running ``make html``. That will generate html files which you can open up locally in your browser.

Reply:
Sure I will check that.